src/test/java/be/seeseemelk/mockbukkit/ServerMockTest.java

```diff
@@ -610,6 +619,28 @@ void testBroadCastMessageWithComponent()
 		playerB.assertSaid(PlainTextComponentSerializer.plainText().serialize(component));
 	}
 
+	@Test
+	void testGetPlayerList()
+	{
+		PlayerMock playerA = server.addPlayer();
+		PlayerMock playerB = server.addPlayer();
+
+		assertThat(server.getPlayerList().getOnlinePlayers(), containsInAnyOrder(playerA, playerB));
+		assertInstanceOf(MockPlayerList.class, server.getPlayerList());
+	}
```

Comment:
Should be moved to a new test class, `MockPlayerListTest`, and renamed to `addPlayers_AddedToList`.